titus-api/src/main/java/com/netflix/titus/api/jobmanager/store/mixin/VolumeMixin.java

```diff
@@ -0,0 +1,21 @@
+package com.netflix.titus.api.jobmanager.store.mixin;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.netflix.titus.api.jobmanager.model.job.SharedContainerVolumeSource;
+
+public abstract class VolumeMixin {
+    @JsonCreator
+    public VolumeMixin(
+            @JsonProperty("name")
```

Comment:
Formatting?

Reply:
I agree it looks weird, but doing the automatic reformatting seems to do nothing for the arguments to the function like this. 

I've manually tried to make it look more like the other files by putting the decorator right next to the argument name, and the reformatter still doesn't care.